resources/zib/zib-InstructionsForUse.xml

```diff
@@ -0,0 +1,269 @@
+  <type value="Dosage" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Dosage" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Dosage">
+      <path value="Dosage" />
+      <short value="InstructionsForUse" />
+      <comment value="This part zib InstructionsForUse is mapped primarily to this Dosage datatype profile. The zib model differs from the FHIR model with regards to the container concepts and their cardinalities: the zibs make a 0..1 reference to InstructionsForUse while in FHIR resources have Dosage inline with a 0..* cardinality. Therefore, the four root elements are mapped differently for better alignment with the zib. Description and RepeatPeriodCyclicalSchedule are provided in extension in the resource. Dosage contains native elements for AdditionalInstructions and RouteOfAdministration hence for these concepts the mapping is placed in the Dosage resulting in duplication of this information when multiple Dosage elements are used. Per InstructionsForUse in the resource, only one route can be captured. Different routes will result in multiple resources that include the spefic InstructionsForUse per route." />
+      <alias value="GebruiksInstructie" />
+      <mapping>
+        <identity value="zib-instructionsforuse-v1.2.1-2020EN" />
+        <map value="NL-CM:9.12.22504" />
+        <comment value="InstructionsForUse" />
+      </mapping>
+    </element>
+    <element id="Dosage.sequence">
+      <path value="Dosage.sequence" />
+      <short value="SequenceNumber" />
+      <definition value="This indicates the sequence of the dosing instructions within the medication agreement." />
+      <alias value="Volgnummer" />
+      <mapping>
+        <identity value="zib-instructionsforuse-v1.2.1-2020EN" />
+        <map value="NL-CM:9.12.22503" />
+        <comment value="SequenceNumber" />
+      </mapping>
+    </element>
+    <element id="Dosage.additionalInstruction">
+      <path value="Dosage.additionalInstruction" />
+      <short value="AdditionalInstructions" />
+      <definition value="The additional instructions contain extra information on the use of the current prescription. &#xD;&#xA;&#xD;&#xA;This includes all instructions for use. The text can come from the original “paper&quot; medication prescription, but can also be generated from the coded information. &#xD;&#xA;&#xD;&#xA;This concept may contain more information than what is structurally coded in the information below, but may not conflict with it. The instructions may not conflict with other components of the request for administration. The instructions can also refer to an existing protocol. The texts in the G-standard that can support this attribute are included in table 362 where the texts from the general practitioner' standard 'NHG-tabel Gebruiksvoorschrift' are included. These texts can be used to structure this concept." />
+      <alias value="AanvullendeInstructie" />
+      <mapping>

```

Comment:
> Why isn't this mapping placed on .additionalInstruction.text?

Because the last couple of sentences in the concept definition mention the use of G-Standard and NHG. These would be better fitted on the CodeableConcept level. Moreover, the MP9 dataset has a ValueSet attached to this concept and made it a code instead of text. I think implementers will understand when to use the .text element without additional comments.  Therefore I think it would be easier to already set this mapping to the CodeableConcept level. Another option would be to map this in the zib-profile to .text and place a mapping in nl-core-profile on the CodeableConcept? 

> I think the phrase "in the information below" is confusing here (actually, it is confusing in the zib as well, but even more so here). I would be in favor of removing this part of the sentence.

Wrong translation of the Dutch I guess, made https://bits.nictiz.nl/browse/ZIB-1492
Oh, the descriptions are still subject to change, the MP dataset is most updated: https://bits.nictiz.nl/browse/MP-156 - I believe zib tickets still need to be created. Which makes me think of #61.

>Reading the description of this concept, I wonder if this wouldn't be a better fit to map onto Dosage.text. Dosage.additionalInstructions are additional instructions, while Dosage.text contains all instructions, just like the zib concept "AdditionalInstructions" ("This includes all instructions for use").

No, I don't think this should be placed on `Dosage.text`. These are the additional instructions, as better described in the 9. 2.0.0 dataset : "De aanvullende instructie bevat een aanvullend uitleg over het gebruik. De aanvullende instructie heeft betrekking op de hele gebruiksinstructie en niet op een individuele dosering."
The Dosage.text contains all information of the InstructionsForUse

Reply:
Removed the strange sentence conform MP-156

Reply:
Forgive me for some tweaking of this text, but I still found it confusing. The Dutch representation makes it clear that this is about additional _guidance_, not additional _information_, and that this guidance may include the coded information and may include additional information. I updated the English comment accordingly.

Reply:
Nice, looks like an improvement.